contrib/test_jenkins.sh

```diff
@@ -1014,6 +1015,63 @@ test_memtrack() {
 	UCX_MEMTRACK_DEST=stdout UCX_HANDLE_ERRORS=none UCX_MEMTRACK_LIMIT=412MB ./test/apps/test_memtrack_limit |& grep -C 100 'reached'
 }
 
+test_namespace() {
+	# Make sure to try to use CMA when possible
+	# Expect fallback on SYSV
+	args="-t ucp_get -s 9999999 -n 5"
+
+	perftest="$ucx_inst/bin/ucx_perftest"
+	info="./src/tools/info/ucx_info"
+
+	echo "==== Running perftest namespace positive tests ===="
+
+	export UCX_LOG_LEVEL="debug"
```

Comment:
why needed?

Reply:
removed the line (intent was to be verbose in the logs to ease any future issue, by dumping lanes)